db/seeds/development/initial_state.js

```diff
@@ -3,25 +3,25 @@ exports.seed = async (knex) => {
   await knex('courses').del();
```

Comment:
I think that it is not necessary

Reply:
okay! I'll erase that line.

It was a copy paste from the users service seed :P